File: src/com/kiwi/kotlin3/java/Person.java
package com.kiwi.kotlin3.java;

import com.kiwi.kotlin3.demo_fun_two.ExtendKt;
import com.kiwi.kotlin3.demo_fun_two.SetTestKt;

import java.util.ArrayList;
import java.util.List;

/**
 * Copyright (C), 2017-2018, XXX有限公司
 * FileName: Person
 * Author:   kiwilss
 * Date:     2018/5/14 14:49
 * Description: person
 * History:
 * <author>          <time>          <version>          <desc>
 * 作者姓名           修改时间           版本号              描述
 */

public class Person {
    private final String name;

    public Person(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }


    public static void main(String[] args) {
        ArrayList<Integer> list = new ArrayList<Integer>();
        list.add(4);
        list.add(4);
        list.add(4);
        String result = SetTestKt.joinToString2(list, ";", "{", "}");
        System.out.println(result);
        char kotlin = ExtendKt.lastChar("kotlin");


        byte b=1;


    }

    <T extends Number> T sum(List<T> list){
        return null;
    }
}

File: src/com/kiwi/jujy/one/class_attribute/Person.kt
package com.kiwi.jujy.one.class_attribute

/**
 * Copyright (C), 2017-2018, XXX有限公司
 * FileName: Person
 * Author:   kiwilss
 * Date:     2018/7/18 10:16
 * Description: person
 * History:
 * <author>          <time>          <version>          <desc>
 * 作者姓名           修改时间           版本号              描述
 */
class Person(val name: String ,var isMarried: Boolean)

fun main(args: Array<String>) {
    val person = Person("Bob", true)

    //改变属性
    person.isMarried=false

    println(person.isMarried)
    println(person.name)





}

